applications/matter_bridge/src/bridge_shell.cpp

```diff
@@ -91,7 +91,7 @@ static int SimulatedBridgedDeviceOnOffWriteHandler(const struct shell *shell, si
 	uint8_t value = strtoul(argv[1], nullptr, 0);
 	chip::EndpointId endpointId = strtoul(argv[2], nullptr, 0);
 
-	DeviceType deviceType{};
+	uint16_t deviceType{};
```

Comment:
in ble_bridged_device_factory.h you've created an alias, and why did you not use it here? Or is it completely different now?

Reply:
It is something different. The `DeviceType` alias here (created few lines above) is a wrapper for helper enum containing currently supported device types for the comparison purposes. Nevertheless user may create a new device type that will not be a part of this enum. So in the end I decided that using uint16_t is more generic (and actually compatible with Matter spec). And the `GetProvider` method takes input uint16_t argument.